app/lib/dns.server.ts

```diff
@@ -302,3 +303,19 @@ export const isValueValid = (type: DnsRecordType, value: string) => {
   // CNAME can be any non-empty string. Let AWS validate it.
   return value.length >= 1;
 };
+
+export const DnsRecordSchema = z
+  .object({
+    subdomain: z.string().min(1),
```

Comment:
Should we do the same type of check you're doing on the `value` below with the subdomain?  We have rules on that too via the `isValidName` function.

Reply:
Oh, that's a good point

Reply:
Can you clarify why you do some of this here, and there rest in the front-end?